### ISSUE TITLE
Reactions via embeds

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-desktop#4918]
This feature request came from a large creator who wants to embed Odysee and at the same time engage their audience. This can drive sign-ups as well.

**Comment by BombardsBodyLanguage** [comment]
A lot of my audience have requested a 'like' button on our website. So having this functionality would be great, and hopefully improve discovery on Odysee.

**Comment by kauffj** [comment]
@BombardsBodyLanguage a LBRY web account would be required -- is that okay with you?

**Comment by BombardsBodyLanguage** [comment]
> @BombardsBodyLanguage a LBRY web account would be required -- is that okay with you?

So you mean my viewers would need a LBRY account in order to fire or slime but *not* required to actually just watch the video?
If so that sounds ok, that's why I have a LBRY link on my site, to encourage people to register to LBRY. I would actually like to incorporate LBRY/Odysee a lot more, but LBRY currently lacks the features/customizability of a website.

Another thing that would be great later down the road particularly once more of my users have signed up to LBRY/Odysee, would be the optional inclusion of comments in the embed code. That should help engagement..

**Comment by kauffj** [comment]
Yes, just account for reactions. Interesting idea to include the comments. Appreciate your feedback, as always!
